_install-and-configure/configuring-opensearch/discovery-gateway-settings.md

```diff
@@ -13,24 +13,23 @@ To learn more about static and dynamic settings, see [Configuring OpenSearch]({{
+| `discovery.type` | String | The cluster formation mode. By default forms a multi-node cluster. Set to `single-node` to run a single-node cluster; this will disable normal discovery. |
+| `cluster.initial_cluster_manager_nodes` | List | Names of cluster-manager-eligible nodes used only at initial cluster bootstrap to elect the first cluster manager. Ignored after the cluster has formed. |
 
-- `discovery.seed_providers` (Static, list): Specifies the types of seed hosts provider to use for obtaining seed node addresses in order to start the discovery process.
-
-- `discovery.type` (Static, string): By default, OpenSearch forms a multi-node cluster. Set `discovery.type` to `single-node` to form a single-node cluster.
-
-- `cluster.initial_cluster_manager_nodes` (Static, list): A list of cluster-manager-eligible nodes used to bootstrap the cluster. 
 
 ## Gateway settings
 
-The local gateway stores cluster state and shard data that is used when a cluster is restarted. OpenSearch supports the following local gateway settings:
-
-- `gateway.recover_after_nodes` (Static, integer): After a full cluster restart, the number of nodes that must join the cluster before recovery can begin.
-
-- `gateway.recover_after_data_nodes` (Static, integer): After a full cluster restart, the number of data nodes that must join the cluster before recovery can begin.
-
-- `gateway.expected_data_nodes` (Static, integer): The number of data nodes expected to exist in the cluster. After the expected number of nodes joins the cluster, recovery of local shards can begin.
+The local gateway stores on-disk cluster state and shard data that is used when a cluster is restarted. The following local gateway **static** settings are supported.
 
-- `gateway.recover_after_time` (Static, time value): The amount of time to wait before starting recovery if the number of data nodes is less than the expected number of data nodes.
+| Setting | Type | Description | Default | Recommendation |
+|:---	|:--- |:---	|:---	|:---	|
+| `gateway.recover_after_nodes` | Integer | The minimum number of total nodes for any role that must be running after a full cluster restart before recovery can begin. | `0` (disabled)—recovery can start as soon as a cluster forms.  | Set to slightly over half of the number of all expected nodes so that the cluster doesn't start recovering with too few nodes. |
+| `gateway.recover_after_data_nodes` | Integer | The minimum number of data nodes that must be running after a full cluster restart before recovery can begin.  | `0` | Set to a significant portion of data nodes---approximately 50–70% of the total data nodes---to avoid premature recovery.  |
+| `gateway.expected_data_nodes` | Integer | The expected number of data nodes in the cluster. When all are present, recovery of local shards can start immediately. | `0` | Set this to the actual number of data nodes in your cluster so that recovery can start immediately once all data nodes are up.  |
```

Comment:
```suggestion
| `gateway.expected_data_nodes` | Integer | The expected number of data nodes in the cluster. When all are present, recovery of local shards can start immediately. | `0` | Set this to the actual number of data nodes in your cluster so that recovery can start immediately once all data nodes are running.  |
```